earthpy/tests/test_clip.py

```diff
@@ -1,96 +1,46 @@
 """Tests for the clip module."""
 
-import numpy as np
 import pytest
```

Comment:
how are the fixtures imported here? ie how does it know where to find those functions if they are not imported? magical?